📸 guildBanAdd - Module Logs

solundev opened the pull request:
- [x] Necessária correções para comando estar operacional e pronto para merge.

solundev committed:
📸 guildBanAdd - Module Logs

- [ ] Necessária correções para comando estar operacional e pronto para merge

oGaabs commented on listeners/guildBanAdd.js:
```diff
@@ -0,0 +1,44 @@
+module.exports = async function onGuildBanAdd(ban, message) {
+	return
+    const guild = this.guilds.cache.get('804575416098488380')
+	if(!ban.guild === guild.id) return;
```
if (ban.guild.id !== guild.id) return

solundev replied:
**Functional v13**

solundev committed:
📢Correções >PuniGC<

- [x] retirado `return` line 6
- [x] if (ban.guild.id !== guild.id) return `// Solved`
